src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -219,9 +219,9 @@ CPU::CPU() {
         } else if (valid_cpu_mapping_table.size() == (unsigned)_processors) {
             return 0;
         } else {
-            std::lock_guard<std::mutex> lock{_cpu_mutex};
             _processors = valid_cpu_mapping_table.size();
             _cpu_mapping_table.swap(valid_cpu_mapping_table);
+            std::lock_guard<std::mutex> lock{_cpu_mutex};
```

Comment:
It's better to add "{}" include the lock and `update_valid_processor_linux` function.

Reply:
updated